django-cloudlaunch/baselaunch/view_helpers.py

```diff
@@ -86,6 +87,13 @@ def get_credentials_from_request(cloud, request):
                     }
         else:
             return {}
+    elif isinstance(cloud, models.GCE):
+        gce_credentials_json = request.META.get('HTTP_CL_GCE_CREDENTIALS_JSON')
```

Comment:
@machristie So I was looking at this a bit. Since an HTTP header can't have any carriage returns or line feeds, I was wondering whether this might pose a problem. However, it looks like JSON data in a header is ok, provided that the json encoder makes sure there are no CR/LFs, which is probably doable. I guess we can test this once the corresponding CloudLaunch UI changes are made.